docs/x509.rst

```diff
@@ -327,6 +391,21 @@ X.509 Extensions
         subordinate CA, but the subordinate CA is not allowed to create
         subordinates with ``ca`` set to true.
 
+.. class:: SubjectAlternativeName
+
+    .. versionadded:: 0.9
+
+    Subject alternative name is an X.509 extension type that provides a list of
+    :class:`GeneralName` objects that provide identities bound to the
+    certificate. The object is iterable to get every :class:`GeneralName`.
+
+    .. method:: get_values_for_type(type)
```

Comment:
This method would not return a list of GeneralName objects. If we want that we could also add a `get_general_names_for_type`.